src/views/utility/utils.js

```diff
@@ -36,6 +36,33 @@ export function setLogoPosition(isStart) {
   }
 }
 
+export function setNavbarLinks(isStart) {
+  //  const navbarStart = getMainLayout().querySelector('vaadin-tabs').shadowRoot.querySelector('.start');
```

Comment:
this commented code probably shouldn't be here ;)

Reply:
deleted